pkg/tagger/tagstore_test.go

```diff
@@ -59,10 +59,8 @@ func (s *StoreTestSuite) TestLookup() {
 		LowCardTags: []string{"tag"},
 	})
 
-	tagsHigh, sourcesHigh, err := s.store.lookup("test", true)
-	assert.Nil(s.T(), err)
-	tagsLow, sourcesLow, err := s.store.lookup("test", false)
-	assert.Nil(s.T(), err)
+	tagsHigh, sourcesHigh := s.store.lookup("test", true)
+	tagsLow, sourcesLow := s.store.lookup("test", false)
```

Comment:
Could you add a simple test case about cache hit/miss state in the unit test?

Reply:
done, thanks